src/main/java/frc/robot/subsystems/MaxWheelModule.java

```diff
@@ -30,6 +30,7 @@ public class MaxWheelModule extends SubsystemBase {
   private SparkClosedLoopController speedController;
   private SparkClosedLoopController angleController;
 
+  // This is an interestingly named variable. Is it just the offset of the module encoder?
```

Comment:
Yeah this is just encoder offset, the template code from rev called it this so I used the same name

Reply:
They are just referring to the angular offset between the 0 of the wheel and parallel to the chassis